internal/overlord/servstate/manager_test.go

```diff
@@ -316,6 +316,62 @@ func (s *S) TestReplanServices(c *C) {
 	s.stopTestServices(c)
 }
 
+func (s *S) TestReplanUpdatesConfig(c *C) {
+	s.startTestServices(c)
+	defer s.stopTestServices(c)
+
+	// Ensure the ServiceManager's config reflects the plan config
+	config := s.manager.Config("test2")
+	c.Assert(config, NotNil)
+	c.Assert(config.OnSuccess, Equals, plan.ActionUnset)
+	c.Assert(config.Summary, Equals, "")
+	command := config.Command
+
+	// Add a layer and override a couple of values
```

Comment:
Nit - comments should end with a "." right?

Reply:
Good call, updated for consistency.